scss/organisms/_banners.scss

```diff
@@ -168,21 +168,6 @@
 	}
 }
 
-.jp-connect-full__button-container {
-	margin: 0;
-	.dops-button.is-primary {
-		background: $green-primary;
-		border-color: $green-secondary;
-		padding-left: rem( 24px );
```

Comment:
Should I bring back this padding?

Reply:
I agree it feels more natural without it.